bin/release.php

```diff
@@ -0,0 +1,158 @@
+<?php
+// phpcs:ignoreFile
+
+/**
+ * This script creates a new branch, updates version numbers and the changelog,
+ * and commits the changes into 2 separate commits. Once this is done, it also
+ * asks whether it should push those changes as a PR on GitHub. It will also
+ * include the changelog into the PR's body, so it can be easily copy/pasted
+ * into the related GitHub release.
+ *
+ * Requirements: GitHub CLI (`brew install gh`).
```

Comment:
It seems like we are not checking the availability of this package ..... so should we check this package at the start of the script to make sure that things go uninterrupted once script is triggered?